Bottom: 8d505aa397edb92ca8a2f03be4f849b02b86f801
Top:    bf913d619d350541750e9e4caa54c6246f4a583e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 14:16:53 +0800

net: core: introduce SOCK_XDP_BUFF

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/include/net/sock.h b/include/net/sock.h
index 4f7c584..e05842b 100644
--- a/include/net/sock.h
+++ b/include/net/sock.h
@@ -783,6 +783,7 @@ enum sock_flags {
 	SOCK_FILTER_LOCKED, /* Filter cannot be changed anymore */
 	SOCK_SELECT_ERR_QUEUE, /* Wake select on error queue */
 	SOCK_RCU_FREE, /* wait rcu grace period in sk_destruct() */
+	SOCK_XDP_BUFF, /* accept a batch of XDP buffs */
 };
 
 #define SK_FLAGS_TIMESTAMP ((1UL << SOCK_TIMESTAMP) | (1UL << SOCK_TIMESTAMPING_RX_SOFTWARE))
